content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -639,25 +639,18 @@ If filing a bug, please include detailed information about how to reproduce the
 * [Relevant logs](https://github.com/kubernetes/community/blob/master/sig-windows/CONTRIBUTING.md#gathering-logs)
 * Tag the issue sig/windows by commenting on the issue with `/sig windows` to bring it to a SIG-Windows member's attention
 
+
 ## {{% heading "whatsnext" %}}
 
-We have a lot of features in our roadmap. An abbreviated high level list is included below, but we encourage you to view our [roadmap project](https://github.com/orgs/kubernetes/projects/8) and help us make Windows support better by [contributing](https://github.com/kubernetes/community/blob/master/sig-windows/).
+### Deployment tools
 
-### Hyper-V isolation
+The kubeadm tool helps you to deploy a Kubernetes cluster, providing the control plane to manage the cluster it, and nodes to run your workloads. [Adding Windows nodes](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/) explains how to deploy Windows nodes to your cluster using kubeadm.
 
-Hyper-V isolation is required to enable the following use cases for Windows containers in Kubernetes:
+The Kubernetes [cluster API](https://cluster-api.sigs.k8s.io/) project also provides means to automate deployment of Windows nodes.
 
-* Hypervisor-based isolation between pods for additional security
-* Backwards compatibility allowing a node to run a newer Windows Server version without requiring containers to be rebuilt
-* Specific CPU/NUMA settings for a pod
-* Memory isolation and reservations
+### Windows distribution channels
 
-Hyper-V isolation support will be added in a later release and will require CRI-Containerd.
+For a detailed explanation of Windows distribution channels see the [Microsoft documentation](https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19).
 
-### Deployment with kubeadm and cluster API
+Information on the different Windows Server servicing channels including their support models can be found at [Windows Server servicing channels](https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19).
```

Comment:
This line and the distribution channels line right above it link to the same page.  I believe "distribution channels" and "servicing channels" mean the same thing in this context and only need to be referenced once.  @immuzz could you confirm?